Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -127,4 +129,52 @@ final class SitterProfileViewModelSaveTests: XCTestCase {
         // Then
         XCTAssertEqual(requestCount, 1)
     }
+
+    func testFirstSaveChangesSitterIsSet() async {
+        // Given
+        MockURLProtocol.requestHandler = { request in
+            let response = try XCTUnwrap(
+                HTTPURLResponse(
+                    url: XCTUnwrap(request.url),
+                    statusCode: 200,
+                    httpVersion: nil,
+                    headerFields: nil
+                )
+            )
+            return (response, nil)
+        }
+
+        // When
+        let sitterIsSetBeforeSave = viewModel.sitterIsSet
+        await viewModel.save()
+        let sitterIsSetAfterSave = viewModel.sitterIsSet
+
+        // Then
+        XCTAssertFalse(sitterIsSetBeforeSave)
+        XCTAssertTrue(sitterIsSetAfterSave)
```

Comment:
Do we really need to create property `sitterIsSetBeforeSave` and `sitterIsSetAfterSave`?
```suggestion
         XCTAssertFalse(viewModel.sitterIsSet)

        // When
        await viewModel.save()

        // Then
        XCTAssertTrue(viewModel.sitterIsSet)
```

Reply:
Changed with 7f84b49

Reply:
The current version of your test is not related to its name.
You check that properties vary, please take a look at my suggestion.

Reply:
Sorry i forget about it, fixed it with 97afb4b